cl/opinion_page/templates/includes/de_list.html

```diff
@@ -171,6 +171,26 @@
                             target="_blank"
                             rel="nofollow">Buy on PACER {% if rd.page_count %}(${{ rd|price }}){% endif %}
                         </a>
+                          {% if request.user.is_authenticated %}
+                          <a href="{% url 'create_prayer' rd.id %}"
+                            class="btn btn-default btn-xs prayer-button {% if rd.prayer_exists or not prayer_eligible %}disabled{% endif %}"
+                            data-document-id="{{ rd.id }}"
+                            {% if rd.prayer_exists %}
+                              title="You have already requested this document."
+                            {% elif not prayer_eligible %}
+                              title="You have reached your daily request limit."
+                            {% endif %}
+                            style="margin-left: {% if rd.page_count %}8px{% else %}44px{% endif %};">
+                            {{rd.prayer_count}} 🙏
+                          </a>
+                        {% else %}
+                          <a href="{% url 'sign-in' %}?next={{ request.path|urlencode }}"
```

Comment:
This directs users to the sign-in page instead of using the modal that the alerts feature employs, because I wasn't quite sure how to incorporate the modal. It works for a MVP, but obviously the modal would be better.

Reply:
I've incorporated the modal to request users to sign in if they're logged out.